v0/destinations/ga4/transform.js

```diff
@@ -41,6 +41,11 @@ const responseBuilder = (message, { Config }) => {
   }
 
   // trim and replace spaces with "_"
+  if (typeof event !== "string") {
```

Comment:
Shouldn't this validation be done very early in the execution?

Reply:
After getting the type of event we are doing these checks first.

Reply:
I see this is needed for all the event types. So, it makes sense to move to the highest level.

Reply:
I think responsebuilder is the best place for this check as for every events we are calling responsebuilder function. And message.event will not available for every events, apart from track for other events we are adding message.event. So it would be better if we do this check in responsebuilder.